v2/charts/azure-service-operator/templates/policy_v1_ poddisruptionbudget_azureserviceoperator-pdb.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  labels:
+    app.kubernetes.io/name: azure-service-operator
+    app.kubernetes.io/version: v2.11.0
```

Comment:
We'll need to update our release docs to include updating this.

Reply:
I think it's done automatically by tooling already

Reply:
it is, see most recent Helm udpate PR: https://github.com/Azure/azure-service-operator/pull/4442/files